connector/fyle_integrations_platform_connector/apis/expenses.py

```diff
@@ -10,7 +10,7 @@ class Expenses(Base):
     """Class for Expenses APIs."""
 
     def get(self, source_account_type: List[str], state: str, last_synced_at: datetime=None,
-        settled_at: datetime=None, filter_credit_expenses: bool=False, last_paid_at=None) -> List[dict]:
+        settled_at: datetime=None,approved_at: datetime=None, filter_credit_expenses: bool=False, last_paid_at=None) -> List[dict]:
```

Comment:
add this at last

Reply:
We would have to do the same change for all the keys. Skipping this for now, we can do this a separate task when we do a cleanup.